From 0f1b21c8dbe9a6b6e880015c82bd25a9ff86ed2d Mon Sep 17 00:00:00 2001
From: Gaurav K Singh <gaurav.k.singh@intel.com>
Date: Tue, 16 Jun 2015 14:41:07 +0530
Subject: [PATCH] FOR_UPSTREAM [VPG]: drm/i915: Set GEM obj ptr to NULL after
 unpin & unreference

In DSI encoder disable sequence,after unpin & unreference
of GEM obj, set gem obj pointer to NULL.

Change-Id: Iba53a9ede2101549f37facdae550017b1453c4d4
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-7482
Signed-off-by: Gaurav K Singh <gaurav.k.singh@intel.com>
---
 drivers/gpu/drm/i915/intel_dsi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/i915/intel_dsi.c b/drivers/gpu/drm/i915/intel_dsi.c
index d6e5388..cc41178 100644
--- a/drivers/gpu/drm/i915/intel_dsi.c
+++ b/drivers/gpu/drm/i915/intel_dsi.c
@@ -763,6 +763,7 @@ static void intel_dsi_post_disable(struct intel_encoder *encoder)
 		mutex_lock(&dev->struct_mutex);
 		drm_gem_object_unreference(&intel_dsi->gem_obj->base);
 		mutex_unlock(&dev->struct_mutex);
+		intel_dsi->gem_obj = NULL;
 	}
 }
 
-- 
1.9.1

